src/test/java/edu/cornell/kfs/module/purap/fixture/CuElectronicInvoiceHelperServiceFixture.java

```diff
@@ -48,9 +70,18 @@ private static String getXMLChunk(){
             xmlChunk.append("timestamp=").
                             append(getCXMLDate(true)).append("\n");
             
-            xmlChunk.append("version=\"1.2.014\" xml:lang=\"en\"\n"); 
-            xmlChunk.append("xmlns=\"http://www.kuali.org/kfs/purap/electronicInvoice\"\n"); 
-            xmlChunk.append("xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\">\n");
+            xmlChunk.append("version=\"1.2.014\" xml:lang=\"en\"\n");
+            
+            if (StringUtils.isNotBlank(xmlnsAttributeName)) {
+                xmlChunk.append(String.format(ATTRIBUTE_NAME_VALUE_FORMAT, xmlnsAttributeName, CuPurapTestConstants.EINVOICE_NAMESPACE_URL));
+                xmlChunk.append("\n");
```

Comment:
this is in a constant kfsconstants.newline

Reply:
Ok, I will update this to use the constant instead.